docs/installation/linux.txt

```diff
@@ -46,7 +46,7 @@ Scroll further below for more OS-specific insructinos.
     cd bigartm
     mkdir build && cd build
     cmake ..
-    make
+    make -j20
```

Comment:
@akashin Why not simply -j?
P.S.: I think someone with 4GB of RAM would be very upset :)